roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -116,7 +116,10 @@ spec:
               value: "logging-{{es_cluster_name}}"
             -
               name: "INSTANCE_RAM"
-              value: "{{openshift_logging_elasticsearch_memory_limit}}"
```

Comment:
If we aren't using this anymore can it be removed from [other places](https://github.com/openshift/openshift-ansible/search?utf8=%E2%9C%93&q=openshift_logging_elasticsearch_memory_limit&type=)?

Reply:
Looks like we already did.

Reply:
in the template we use `es_memory_limit` because the template is use for both app and ops ES deployments.